src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.FSharp.xaml

```diff
@@ -33,6 +35,32 @@
                 HelpUrl="https://docs.microsoft.com/dotnet/fsharp/language-reference/compiler-options"
                 Category="General" />
 
+  <EnumProperty Name="FSharpPreferNetFrameworkTools"
+              DisplayName="Framework tools"
+              Description="Compiler to use when building the project."
```

Comment:
We use complete sentences for descriptions.

```suggestion
              Description="Specifies which compiler to use when building the project."
```

A question on this property. The name says "prefer" which makes it sound like it'll use the .NET Framework compiler if it's available. This metadata will make the property appear less like a preference and more like an absolute directive. What happens if I select ".NET Framework" in the UI, but the .NET Framework compiler is not available? Could the user be surprised here?

Reply:
If the .NET Framework is not available, but selected, we will ignore the property, and chose the SDK compiler.

Reply:
In that case I think this should be a checkbox with text "Prefer the .NET Framework compiler when available."

Right now it seems like a guarantee that the specified compiler will be used.

Reply:
@KevinRansom are you OK with having it as a boolean (checkbox)? Saying "Prefer .NET Framework version of the compiler when available".